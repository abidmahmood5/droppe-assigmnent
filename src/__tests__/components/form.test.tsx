import React from 'react';
import renderer from 'react-test-renderer';
import Form from '../../components/form';

it('creates a form', () => {
  const handleSubmit = jest.fn();

  const formComponent = renderer.create(
    <Form on-submit={handleSubmit} />
  ).toJSON();

  expect(formComponent).toMatchSnapshot();
});
