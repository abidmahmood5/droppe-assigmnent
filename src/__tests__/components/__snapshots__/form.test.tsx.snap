// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a form 1`] = `
<form
  className="form"
  onSubmit={[Function]}
>
  <span
    className="label"
  >
    Product title: *
  </span>
  <input
    className="input"
    defaultValue=""
    placeholder="Title..."
  />
  <span
    className="label"
  >
    Product details: *
  </span>
  <input
    className="input"
    defaultValue=""
    placeholder="Price..."
  />
  <textarea
    className="textarea"
    defaultValue=""
    placeholder="Start typing product description here..."
  />
  <button
    className="button"
  >
    Add a product
  </button>
</form>
`;
